pkg/db/db_test.go

```diff
@@ -2596,39 +2596,12 @@ func TestReadWriteOverviewCache(t *testing.T) {
 										Argocd:           &api.EnvironmentConfig_ArgoCD{},
 										EnvironmentGroup: &dev,
 									},
-									//Applications: map[string]*api.Environment_Application{
```

Comment:
Read and write overview cache, we can simply ignore the information that was deleted